ui/components/component-library/text/text.constants.js

```diff
@@ -3,3 +3,9 @@ export const TEXT_DIRECTIONS = {
   RIGHT_TO_LEFT: 'rtl',
   AUTO: 'auto',
 };
+
+/**
+ * The INVISIBLE_CHARACTER is a very useful tool if you want to make sure a line of text
+ * takes up vertical space even if it's empty.
+ */
+export const INVISIBLE_CHARACTER = '\u200d';
```

Comment:
I don't have any experience using invisible characters? Wouldn't we just want the html node to not be rendered? I did bring this up with @Akatori-Design and he sad that the movement/jump of the form elements when there is an error is the expected behaviour. @garrettbear how do you feel about this?

Reply:
I guess it would still jump if the error message is more than one line?

Reply:
`\u200B` and `\u200D` appear to function the same way, but the `\u200B` description says "this character is intended for invisible word separation and for line break control," so I think you're right that it's a better choice.

Yeah, the error message is always at least one line high, and it still jumps if the error message is more than one line.  The INVISIBLE_CHARACTER is I think a better alternative to what I was doing before with `marginTop={warning ? 0 : 6}`

Reply:
When I first built the form components I did try to account for the jumping but @Akatori-Design said that this was the intended behaviour 


https://user-images.githubusercontent.com/8112138/222871266-78b353da-c9c3-49c4-ac78-572e6cad46b7.mov

Reply:
Okay I made it jump, I don't really love it, but oh well

Reply:
Thanks. I'll bring it up in our next design system sync and if the pattern changes we can do it for all error messages in a different PR